docs/index.adoc

```diff
@@ -394,25 +394,87 @@ creation of a new active wallet regardless of age.
 |Yes
 |`1 week`
 
-|movedFundsSweepTimeoutSlashingAmount
-|The amount of stake to slash if the wallet does not move its funds in time.
-|Yes
-|`100% of min-stake`
-
 |movedFundsSweepTimeoutNotifierRewardMultiplier
 |The the percentage of the slashed stake that the notifier receives as a reward.
 |Yes
 |`5%`
 
 |walletMaxBtcTransfer
-|The threshold at which we try to divide up a closing wallet into multiple target wallets
+|The amount of satoshis at which we try to divide up a closing wallet into multiple target wallets.
 |Yes
-|`50 BTC`
+|`10e8`
 
 |walletClosingPeriod
 |The amount of time the wallet remains in the `Closing` state before it is closed.
 |Yes
-|`3 days`
+|`40 days`
+
+|walletClosureMinBtcBalance
+|The minimum amount of satoshis at which a wallet is eligible to begin closing.
+|Yes
+|`5 * 1e7`
+
+|walletMaxAge
+|The age at which a wallet is eligible to begin closing.
+|Yes
+|`26 weeks`
+
+|movingFundsTxMaxTotalFee
+|The max amount of satoshis that the wallet is allowed to pay miners for a moving funds transaction.
+|Yes
+|`10000 satoshis`
+
+|movingFundsDustThreshold
+|The minimum amount of satoshis that we move from wallet to wallet when closing a wallet.
+|Yes
+|`20000 satoshis`
+
+|movingFundsTimeoutResetDelay
+|The amount of time operators must wait to request a timeout reset on moving funds when there is no active wallet.
+|Yes
+|`6 days`
+
+|movingFundsTimeout
+|The amount of time operators have to move their wallet's funds to another live wallet(s).
```

Comment:
```suggestion
|The amount of time the operators have to move their wallet's funds to another live wallet(s).
```
It sounds better to me, but I'm not 100% sure if this suggestion is right :P

Reply:
[54b7a3a](https://github.com/keep-network/tbtc-v2/pull/346/commits/54b7a3aa103d9dfe4e3cce4542eb7b703710ba88)